ui-support-persistence/src/main/java/org/dcsa/uisupport/persistence/entity/PortWithTimezone.java

```diff
@@ -0,0 +1,46 @@
+package org.dcsa.uisupport.persistence.entity;
+
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.PrimaryKeyJoinColumn;
+import javax.persistence.SecondaryTable;
+import javax.persistence.Table;
+
+/**
+ * Joins un_location and port_timezone.
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PRIVATE)
+@Entity
+@Table(name = "un_location")
+@org.hibernate.annotations.Table(appliesTo = "port_timezone", optional = false) // forces an inner join instead of outer join
```

Comment:
`LEFT OUTER JOIN` is fine for this case. The UI copes with missing timezones and it is better for us to show the port but note that we are missing the correct timezone than not show the port at all.